### PR TITLE
Made a start on the EasyPID Readme

### PR DESCRIPTION
**Pull request by anagoessens**
Still needs quite some work.

### CODE REVIEW

File: apps/easypid/README.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+- [OpenID for Verifiable Credential Issuance - ID 1 / Draft 13](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0-ID1.html)
+- [OpenID for Verifiable Presentations - Draft 20](https://openid.net/specs/openid-4-verifiable-presentations-1_0-20.html)
+- [SD-JWT VC - Draft 3](https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-03.html)
+- [Self-Issued OpenID Provider V2 - Draft 13](https://openid.net/specs/openid-connect-self-issued-v2-1_0-13.html)
+- [ISO 18013-5](https://www.iso.org/standard/69084.html)
+- [ISO/IEC TS 23220-3 - WD13](https://www.iso.org/standard/86783.html)
+- [ISO/IEC TS 18013-7 DTS Ballot Text](https://www.iso.org/standard/82772.html)
+- [High Assurance Interop Profile - Draft 0](https://openid.net/specs/openid4vc-high-assurance-interoperability-profile-sd-jwt-vc-1_0-00.html)
+- ❌ [OpenID Federation - Draft 34](https://openid.net/specs/openid-federation-1_0-34.html)
+- [OAuth 2.0 Attestation-Based Client Authentication](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-attestation-based-client-auth-01)
```
Comment (TimoGlastra) [inline apps/easypid/README.md:90]: not implemented

File: apps/easypid/README.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+The following components were used.
+- [Credo](https://github.com/openwallet-foundation/credo-ts)
+
+The following standards and specifications were implemented.
+- [OpenID for Verifiable Credential Issuance - ID 1 / Draft 13](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0-ID1.html)
+- [OpenID for Verifiable Presentations - Draft 20](https://openid.net/specs/openid-4-verifiable-presentations-1_0-20.html)
+- [SD-JWT VC - Draft 3](https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-03.html)
+- [Self-Issued OpenID Provider V2 - Draft 13](https://openid.net/specs/openid-connect-self-issued-v2-1_0-13.html)
+- [ISO 18013-5](https://www.iso.org/standard/69084.html)
+- [ISO/IEC TS 23220-3 - WD13](https://www.iso.org/standard/86783.html)
```
Comment (TimoGlastra) [inline apps/easypid/README.md:86]: @auer-martin any idea if we support this spec?

File: apps/easypid/README.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+- [ISO/IEC TS 18013-7 DTS Ballot Text](https://www.iso.org/standard/82772.html)
```
Comment (TimoGlastra) [inline apps/easypid/README.md:87]: @auer-martin i think we support this except for JARM right?

File: apps/easypid/README.md (hunk 1 above)
Comment (anagoessens) [reply apps/easypid/README.md:90]: I thought we could iterate on the readme so could use ✅, 🚧, and ❌ to indicate that we will in the future implement others still. OpeniDFed was the only one I know for sure

File: apps/easypid/README.md (hunk 3 above)
Comment (auer-martin) [reply apps/easypid/README.md:87]: yes jarm and the issuersigned

**Pull request merged**